pysmt/formula.py

```diff
@@ -497,6 +498,191 @@ def Max(self, *args):
             h = len(exprs) // 2
             return self.Max(self.Max(exprs[0:h]), self.Max(exprs[h:]))
 
+    # BitVectors
+    def BV(self, value, width=None):
+        """Return a constant of type BitVector."""
+
+        try:
+            if value.startswith("#b"):
+                value = value[2:]
+            if value[0] in ["0", "1"]:
+                width = len(value)
+                value = int(value, 2)
+        except ValueError:
```

Comment:
OK, python goeas for EAFP, but this is quite cryptic. why not checking if value is a string instead of catching attribute error?